### PR TITLE
New version

### PR DESCRIPTION
**Pull request by MaxElectron**
Made a first working iteration of the project.

Guide on playing:
- Download the python file
- Place it in the same directory where you the save file to be
- Start the game and interact with it through console interface by typing commands
- First you'll need to go through pet creation
- After that you'll be able to play
- There is a "Help" command in game
- Train your cat and take care of it

New features:
- A cat pet
- Pet creation
- Pet status
- Basic commands
- Idle handling
- Ascii graphics
- Main gameloop

### CODE REVIEW

**Comment by MaxElectron** [comment]
So, about the "only one file" thing: I'm going to fix it this weekend by putting all bulky print parts into functions and splitting file into several parts by function types. If you have any other concerns feel free to point them out, even though the "one file" problem is not fixed yet, so I can also fix those. Sorry about being late on adding you as a reviewer, it was a busy middle of the week, hadn't been checking on the GitHub for a few days.

**Comment by MaxElectron** [comment]
Updated features:
 - Fixed issue with code beeing inside a single file
 - Added a comprehensive README file

Comment:

 - _So, that's basically ready for review, hope it's good, not really planning to further develop this project, but will fix all the complaints you'll have._

File: README.txt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+How to install python and python packages:
+
+- Go get Ubuntu
+- Open the Terminal
+- Do following commands:
+      - sudo apt update
+      - sudo apt upgrade
+      - sudo apt install python3-pip
+      - sudo apt install build-essential libssl-dev libffi-dev python3-dev
+- Launch the file with python
```
Comment (chupolino17) [inline README.txt:34]: Стоит делать readme все же в md, тогда он будет поддерживать markdown

**Review by chupolino17 — APPROVED** [review]
Не оч прикольно из за того, что ввод команд работает через раз, а так ничего

**Pull request merged**